packages/grafana-toolkit/src/cli/utils/githubClient.ts

```diff
@@ -36,6 +36,17 @@ class GithubClient {
   private createClient(clientConfig: AxiosRequestConfig) {
     return axios.create(clientConfig);
   }
+
+  public async getIssues(state = 'open', milestone: string, perPage = 100, ...labels: string[]) {
```

Comment:
can you remove public keyword here, it's causing lint error

Reply:
@torkelo Yes, sure. Pushed it just for an example of implementation. Could you please reply me in the referenced issue?

Reply:
@Russiancold I left my comments on the issue itself. Thank you again for your contribution!